### ISSUE TITLE
Enhancement - better message for lack of servers

### ISSUE THREAD

**Issue by Cinderella-Man** [neumino/rethinkdbdash#250]
We are in process of setting up Ansible auto-deployment of our infrastructure and some script didn't fill the servers' IP addresses and we got back following (not very descriptive) error message:

```
/myproject/node_modules/rethinkdbdash/lib/dequeue.js:9
  this.buffer = new Array(size);
                ^

RangeError: Invalid array length
    at new Dequeue (/myproject/node_modules/rethinkdbdash/lib/dequeue.js:9:17)
    at new Pool (/myproject/node_modules/rethinkdbdash/lib/pool.js:34:16)
    at new PoolMaster (/myproject/node_modules/rethinkdbdash/lib/pool_master.js:62:12)
    at r.createPools (/myproject/node_modules/rethinkdbdash/lib/index.js:85:22)
    at main (/myproject/node_modules/rethinkdbdash/lib/index.js:528:34)
    at Object.<anonymous> (/myproject/build/RethinkDb/init.js:30:33)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:159:18)
    at node.js:444:3
```

code to replicate:

```
var r = require("rethinkdbdash")({
  "servers": [],
  "db" : "itheon",
  "replicas": 5
});
```

This could probably be updated to be more descriptive.

**Comment by neumino** [comment]
Sorry for the late answer. This look like a bug to me. I'm not sure we should fallback to localhost:28015 in this case. We should probably just throw a human friendly error.

**Comment by neumino** [comment]
Fixed in 2.3.6
